omnij-jsonrpc/src/main/java/foundation/omni/rpc/OmniClient.java

```diff
@@ -311,6 +311,22 @@ public Sha256Hash omniSendAll(Address fromAddress, Address toAddress, Ecosystem
         return send("omni_sendall", Sha256Hash.class, fromAddress, toAddress, ecosystem);
     }
 
+    /**
+     * Creates and broadcasts a "send to many" transaction.
+     *
+     * @param fromAddress The address to spent from
+     * @param currency    The identifier of the token to distribute
+     * @param mapping     The receiving addresses and amounts
```

Comment:
Why is this a `List` of `Maps`. What is the significance of each position in the list?

Reply:
I think the answer here is that each `Map` is actually a "tuple" -- meaning an address/amount pair and that it would be considered invalid for a Map to contain more than one entry.

Reply:
Here is an example of the Omni Core RPC:

```
omni_sendtomany "2N9UopnCBgbC6wAjMaiqrBZjmgNFiZccY7C" 3 '[{"address": "mnM5bS3qQTxZSHkgpp2LDcKYL5thWtqCD6", "amount": "3"}, {"address": "2N9UopnCBgbC6wAjMaiqrBZjmgNFiZccY7C", "amount": "5"}]'
```